### PR TITLE
482 dashboard icon

### PR DESCRIPTION
**Pull request by matthopson**

### CODE REVIEW

File: web-client/src/views/CaseListPetitioner.jsx [hunk 1]
```diff
@@ -38,6 +39,7 @@ export const CaseListPetitioner = connect(
           >
             <thead>
               <tr>
+                <th></th>
```
Comment (kkoskelin) [inline web-client/src/views/CaseListPetitioner.jsx]: For a11y, could we add something like `<span className="usa-sr-only">Lead Case Indicator</span>` ?

File: web-client/src/views/CaseListPetitioner.jsx [hunk 2]
```diff
@@ -46,6 +48,15 @@ export const CaseListPetitioner = connect(
             <tbody>
               {formattedCases.map(item => (
                 <tr key={item.docketNumber}>
+                  <td>
+                    {item.isLeadCase && (
+                      <FontAwesomeIcon
```
Comment (kkoskelin) [inline web-client/src/views/CaseListPetitioner.jsx]: `<span className="usa-sr-only">Lead Case</span>` as a companion label for a11y?

File: web-client/src/views/CaseListPractitioner.jsx [hunk 3]
```diff
@@ -18,6 +19,7 @@ export const CaseListPractitioner = connect(
         <table className="usa-table responsive-table dashboard" id="case-list">
           <thead>
             <tr>
+              <th></th>
```
Comment (kkoskelin) [inline web-client/src/views/CaseListPractitioner.jsx]: Same a11y request here, please.

File: web-client/src/views/CaseListRespondent.jsx [hunk 4]
```diff
@@ -16,6 +17,7 @@ export const CaseListRespondent = connect(
         <table className="usa-table responsive-table dashboard" id="case-list">
           <thead>
             <tr>
+              <th></th>
```
Comment (kkoskelin) [inline web-client/src/views/CaseListRespondent.jsx]: a11y. you get it. :D

**Review by kkoskelin — APPROVED** [review]
Looks good. I prefer, when we can, to add the a11y things like screenreader labels.  If not this PR, the next?

**Review by ericsorenson — APPROVED** [review]

**Review by sutt0n — APPROVED** [review]

**Pull request merged**